x-pack/legacy/plugins/apm/public/components/app/ServiceNodeMetrics/index.tsx

```diff
@@ -117,6 +117,23 @@ export function ServiceNodeMetrics() {
         </EuiCallOut>
       ) : (
         <EuiFlexGroup gutterSize="xl">
+          <EuiFlexItem grow={false}>
+            <EuiStat
+              titleSize="s"
+              isLoading={isLoading}
```

Comment:
Having this loading indicator feels a little off since `serviceName` is always available - it never needs to be loaded.
OTOH I can see why it perhaps looks odd if some of the flex items show up immediately and the others are loaded afterwards. If that's the case 👍 Just wanted to point it out in case it was an oversight :)

Reply:
I agree with @sqren here about showing the `service.name` instantly and letting the other values load in when ready.